src/vector/init.tsx

```diff
@@ -124,13 +126,33 @@ export async function showIncompatibleBrowser(onAccept: () => void): Promise<voi
     );
 }
 
+/**
+ * @deprecated in favour of the plugin system
+ */
 export async function loadModules(): Promise<void> {
     const { INSTALLED_MODULES } = await import("../modules.js");
     for (const InstalledModule of INSTALLED_MODULES) {
         ModuleRunner.instance.registerModule((api) => new InstalledModule(api));
     }
 }
 
+export async function loadPlugins(): Promise<void> {
+    // add React to the global namespace, to make them easier to access via the console
+    // this incidentally means we can forget our React imports in JSX files without penalty.
+    window.React = React;
```

Comment:
Dows this mean this is actually necessary for plugins?

Reply:
It is in the current direction, otherwise each plugin that wants to render anything would need to bundle its own React version and track the same React version EW is using. Instead it can just reuse it.

Reply:
Cool, could that go in the comment?